docs/Release_Notes.md

```diff
@@ -1,10 +1,10 @@
-GFS V16.3.18 RELEASE NOTES
+GFS V16.3.19 RELEASE NOTES
 
 -------
 PRELUDE
 -------
 
-The WAFS blending 0.25 job is updated to increase the wait time for UKMet data from 900 to 1500 seconds. Additional error handling is also added.
+PlanetiQ RO data has added noise and is placed into monitor mode.
```

Comment:
The EMC DA team plans to evaluate PlanetiQ RO data quality in a cycled test.  Pending neutral to positive results a subsequent RFC will be submitted to assimilate the data.  It would be good to modify the statement I provided earlier to reflect this.  

Suggested rewording below:

> PlanetiQ RO data has added noise and is placed into monitor mode until its impact on the cycled analysis forecast system can be assessed.

Reply:
Done @ c23ef5ab. Thanks!